user/installation.rst

```diff
@@ -4,65 +4,55 @@ Installation
 The functionality of ``colcon`` is split over multiple Python packages.
 The package ``colcon-core`` provides the command line tool ``colcon`` itself as well as few fundamental extensions.
 Additional functionality is provided by separate packages, e.g. ``colcon-cmake`` adds support for packages which use `CMake <https://cmake.org/>`_.
-In the following a set of common ``colcon`` packages is being installed.
+The following instructions install a set of common ``colcon`` packages.
 
-The most common way of installation for users is to the Python package manager ``pip``.
+Using Debian packages
+---------------------
 
-Using pip on any platform
--------------------------
-
-The following assumes that you are using a virtual environment with Python 3.5 or higher.
-If you want to install the packages globally it might be necessary to invoke ``pip3`` instead of ``pip`` and require ``sudo``.
-
-.. code-block:: bash
+On platforms which support Debian packages using those is preferred since they will be updated using ``apt`` together with other system packages.
 
-    $ pip install -U colcon-common-extensions
+The Debian packages are currently hosted in apt repositories from the ROS project.
+You can choose either of the two following apt repositories.
 
-.. note::
+* `ROS 1 repository <http://wiki.ros.org/Installation/Ubuntu#Installation.2BAC8-Ubuntu.2BAC8-Sources-4.Setup_your_sources.list>`_
 
-    The package ``colcon-common-extensions`` doesn't contain any functionality itself but only depends on a set of other packages (see `setup.cfg <https://github.com/colcon/colcon-common-extensions/blob/master/setup.cfg>`_).
+  .. code-block:: bash
 
-.. note::
+      $ sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.list'
+      $ sudo apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 421C365BD9FF1F717815A3895523BAEEB01FA116
 
-    You can find a list of released packages on `PyPI <https://pypi.org/search/?q=colcon>`_ using the keyword ``colcon``.
+* `ROS 2 repository <https://github.com/ros2/ros2/wiki/Linux-Install-Debians#setup-sources>`_
 
-Shell specific packages:
-~~~~~~~~~~~~~~~~~~~~~~~~
+  .. code-block:: bash
 
-Bash
-^^^^
+      $ sudo sh -c 'echo "deb [arch=amd64,arm64] http://repo.ros2.org/ubuntu/main `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
+      $ curl http://repo.ros2.org/repos.key | sudo apt-key add -
```

Comment:
I know this is copied from the instructions but we should consider using a keyserver instead of piping the result of curl (can be done in a follow-up once it's adressed in other places like CI and installation instructions)

Reply:
Once upstream changes this can be synced :+1: